tests/docker/docker-compose.yml

```diff
@@ -42,5 +42,6 @@ services:
     - minio
     volumes:
     - '${BUILD_ROOT}:${BUILD_ROOT}'
+    - '${CLUSTERS_HOST_PATH:-$BUILD_ROOT/var}:/host_var'
```

Comment:
when `CLUSTERS_HOST_PATH` is undefined, this will end up mounting the same folder for all containers, so all redpanda instances will be writing to the same folder, causing issues.

IMO there’s no need to have `/host_var` as an indirection point. The containers can bind-mount `/mnt/scratch` regardless of whether ducktape tests write to it or not:

```suggestion
    - /mnt/scratch:/mnt/scratch
```

we could also make this a variable (e.g. `DUCKTAPE_SCRATCH_STATE_DIR` or something like that)

Reply:
The default behaviour of the `CLUSTERS_HOST_PATH` is kind of useless, I just don't want this to fail if it is not defined. The `host_var` is not used in the default case, it is only used [here](https://github.com/redpanda-data/vtools/pull/618/files#diff-0af5f43de37b92f214f432df31cec26fb3001cbc75f152cb0cf5f6010bfec2cdR157). The ducktape will still behave as we have it now, so it won't cause any additional problems. Specifically, it will still be holding redpanda data under `/var/lib/redpanda`, which is what we currently have. A good proof for that is the tests in this PR that ran ok. Although, it is confusing, so I'll try to make it simpler

Reply:
fixed: added /mnt/scratch as default for both host and container paths